autosklearn/automl.py

```diff
@@ -1554,65 +1554,55 @@ def _load_models(self):
             self.ensemble_ = None
 
         # If no ensemble is loaded, try to get the best performing model
-        if not self.ensemble_:
-            self.ensemble_ = self._load_best_individual_model()
-
-        if self.ensemble_:
-            identifiers = self.ensemble_.get_selected_model_identifiers()
-            self.models_ = self._backend.load_models_by_identifiers(identifiers)
-
-            if self._resampling_strategy in ("cv", "cv-iterative-fit"):
-                self.cv_models_ = self._backend.load_cv_models_by_identifiers(
-                    identifiers
+        if (
```

Comment:
Can this be simplified and just checking whether ensemble_size == 0?

Reply:
I am not sure, so I'd rather keep it.

Reply:
Probably not, I just removed this statement.